pkg/executor/types.go

```diff
@@ -23,24 +23,42 @@ import (
 const (
 	// ClusterLabel is a label placed on every created statefulset, pod, and
 	// service, with a value of the KubeDirectorCluster CR name.
-	ClusterLabel = "kubedirectorcluster"
+	ClusterLabel = shared.KdDomainBase + "/kdcluster"
+	// ClusterAppLabel is a label placed on every created statefulset, pod,
+	// and service, with a value of the KubeDirectorApp CR name.
+	ClusterAppLabel = shared.KdDomainBase + "/kdapp"
+	// ClusterAppCatalogLabel is a label placed on every created statefulset,
+	// pod, and service, with a value "local" or "system" appropriately.
+	ClusterAppCatalogLabel = shared.KdDomainBase + "/appCatalog"
 	// ClusterRoleLabel is a label placed on every created pod, and
 	// (non-headless) service, with a value of the relevant role ID.
-	ClusterRoleLabel     = "role"
-	headlessServiceLabel = shared.KdDomainBase + "/" + "headless"
-	statefulSetPodLabel  = "statefulset.kubernetes.io/pod-name"
-	storageClassName     = "volume.beta.kubernetes.io/storage-class"
-	// AppContainerName is the name of kubedirector app containers
-	AppContainerName    = "app"
-	pvcName             = "pvc"
-	headlessServiceName = "svc"
-	initContainerName   = "init"
-	execShell           = "bash"
-	configMetaFile      = "/etc/guestconfig/configmeta.json"
-	cgroupFSVolume      = "/sys/fs/cgroup"
-	systemdFSVolume     = "/sys/fs/cgroup/systemd"
-	tmpFSVolSize        = "20Gi"
-	kubedirectorInit    = "/etc/kubedirector.init"
+	ClusterRoleLabel = shared.KdDomainBase + "/role"
+	// HeadlessServiceLabel is a label placed on the statefulset and pods.
+	// Used in a selector on the headless service.
+	HeadlessServiceLabel = shared.KdDomainBase + "/headless"
+
+	// ClusterAppAnnotation is an annotation placed on every created
+	// statefulset, pod, and service, with a value of the KubeDirectorApp's
+	// spec.label.name.
+	ClusterAppAnnotation = shared.KdDomainBase + "/kdapp-prettyName"
```

Comment:
?

Reply:
spec.label.name from the kdapp. Can't store it as a label value because it is an arbitrary string.